contextualized/regression.py

```diff
@@ -358,10 +358,13 @@ def forward(self, C, T):
 
 
 class ContextualizedRegressionBase(pl.LightningModule):
-    def __init__(self, *args, learning_rate=1e-3, link_fn=linear_fn(), **kwargs):
+    def __init__(self, learning_rate=1e-3, link_fn=linear_fn(), loss_fn=MSE,
+                 sample_model_regularizer=zero_vector, *args, **kwargs):
         super().__init__()
-        self.link_fn = link_fn
         self.learning_rate = learning_rate
+        self.link_fn = link_fn
+        self.loss_fn = loss_fn
+        self.sample_model_regularizer = sample_model_regularizer
```

Comment:
Should be `model_regularizer` based on the current convention of metamodels and models.